src/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -19,6 +19,18 @@ const render = (props = {}, tabProps = {}) =>
   );
 
 describe('<Tabs />', () => {
+  it('works with a single child tab', () => {
+    const wrapper = shallow(
+      <Tabs>
+        <Tab id="tab-1" title="Tab 1">
+          <Box>Tab 1 Content</Box>
+        </Tab>
+      </Tabs>
+    );
+
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
A test to enforce acceptance of a single-child Tab.  This wouldn't previously have compiled.